src/views/_proxied-dot-io/product-docs-landing/types.ts

```diff
@@ -0,0 +1,15 @@
+import { Products } from '@hashicorp/platform-product-meta'
+import { FeaturedCardProps } from './components/featured-card/types'
+import { UseCaseCardProps } from './components/use-case-card/types'
+
+export interface ProductDocsLandingProps {
```

Comment:
This type doesn't do tons since we're importing from JSON... but figured worth articulating anyways.

Reply:
✅ added! great suggestion 💖